src/witness/yamlWitness.ml

```diff
@@ -124,6 +152,11 @@ struct
 
   module NH = BatHashtbl.Make (Node)
   module WitnessInvariant = WitnessUtil.Invariant (File) (Cfg)
+  module FMap = Prelude.Hashtbl.Make (CilType.Fundec)
+  module FCMap = Prelude.Hashtbl.Make (Printable.Prod (CilType.Fundec) (Spec.C))
```

Comment:
Why `Prelude`?

Reply:
Just a noisy way of using `BatHashbtl`. I changed it.